src/styles/index.scss

```diff
@@ -1,3 +1,7 @@
+@import 'src/styles/utils/variables';
+@import 'src/styles/blocks/card';
+@import 'src/styles/blocks/stars';
```

Comment:
Ensure that the file `src/styles/blocks/stars.scss` exists and is correctly imported.